static/app/views/issueList/actions/resolveActions.tsx

```diff
@@ -1,14 +1,23 @@
 import ResolveActions from 'sentry/components/actions/resolve';
-import {Organization} from 'sentry/types';
+import {Organization, Release} from 'sentry/types';
 
-import {ConfirmAction} from './utils';
+import {ConfirmAction, getConfirm, getLabel} from './utils';
 
 type Props = {
   anySelected: boolean;
   onShouldConfirm: (action: ConfirmAction) => boolean;
   onUpdate: (data?: any) => void;
   orgSlug: Organization['slug'];
-  params: any;
+  params: {
```

Comment:
Added types here since it used to be `any` and caused me to miss one of the uses of `confirm` when I changed the type signature.